pkg/fakerp/server.go

```diff
@@ -109,13 +122,18 @@ func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, resp, http.StatusInternalServerError)
 		return
 	}
+
+	// delete dns records
+	err = DeleteOCPDNS(ctx, s.conf.SubscriptionID, s.conf.ResourceGroup, s.conf.DnsResourceGroup, s.conf.DnsDomain, config)
+	if err != nil {
+		s.log.Fatal(err)
```

Comment:
Can't be `s.log.Fatal(err)` but similar to the rest of the error handling we do in the server.